src/Weapon.js

```diff
@@ -881,16 +885,21 @@ class Weapon {
     if (this.autoExpandBulletsGroup) {
       bullet = this.bullets.getFirstDead(true, fromX, fromY, this.bulletKey, this.bulletFrame);
 
-      bullet.bulletManager = this;
+      bullet.data.bulletManager = this;
     } else {
       bullet = this.bullets.getFirstDead(false);
     }
 
+    console.log(`got bullet: ${bullet ? bullet.bulletID : 'none, :('}`)
+
     if (bullet) {
       bullet.body.reset(fromX, fromY);
 
+      // unclear if we actually need to set this to active here or if this
+      // should be the bullet itself
       this.active = true;
       this.visible = true;
+      bullet.active = true;
```

Comment:
Look like it probably should be something like this.
```js
bullet.active = true;
bullet.visible = true; // this actually makes the bullets render 😀
```
Not sure if the ones with `this` are needed or not, they might be used to keep track of something.
But I'm pretty sure the ones with `bullet` were missing, because now the bullets actually render (which makes sense, if they were `visible = false` the whole time)

Reply:
Commited a fix!